lib/keysim.js

```diff
@@ -401,6 +401,9 @@ export class Keyboard {
         return true;
 
       default:
+        if (target.designMode.toLowerCase === 'on' || target.isContentEditable) {
```

Comment:
This is not how `designMode` should be checked. `designMode` should be checked on the `document` property of the provided element.